test/performance/broker-kafka/100-broker-perf-setup.yaml

```diff
@@ -0,0 +1,228 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: perf-eventing
+
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-mako
+  namespace: perf-eventing
+
+data:
+  _example: |
+    ################################
+    #                              #
+    #    EXAMPLE CONFIGURATION     #
+    #                              #
+    ################################
+
+    # This block is not actually functional configuration,
+    # but serves to illustrate the available configuration
+    # options and document them in a way that is accessible
+    # to users that `kubectl edit` this config map.
+    #
+    # These sample configuration options may be copied out of
+    # this example block and unindented to be in the data block
+    # to actually change the configuration.
+
+    # The Mako environment in which we are running.
+    # Only our performance automation should run in "prod", but
+    # there should be a "dev" environment with a fairly broad
+    # write ACL.  Users can also develop against custom configurations
+    # by adding `foo.config` under their benchmark's kodata directory.
+    environment: dev
+
+    # Additional tags to tag the runs. These tags are added
+    # to the list that the binary itself publishes (Kubernetes version, etc).
+    # It is a comma separated list of tags.
+    additionalTags: "key=value,absolute"
+
+---
+
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Broker
+metadata:
+  name: kafka-test-broker
+  namespace: perf-eventing
+spec:
+  channelTemplateSpec:
+    apiVersion: messaging.knative.dev/v1alpha1
+    kind: KafkaChannel
+
```

Comment:
we want to capture _specific_ for the channel ? e.g `partitions`  ?  sure, could be added later!

Reply:
I think the purpose of these scripts are to serve as a basis to run the tests. Then, based on what you are searching, you tune it :)

Reply:
perfect!